### PR TITLE
WMS ID#7321  Get Started with MySQL-HeatWave and Feel the Difference.

### PR DESCRIPTION
**Pull request by jitender-git**
The workshop was already approved and merged into master library however while doing QA with WMS team, they suggested some minor changes (like adding #Acknowledgement). Hence request you to please approve so that it can be merged and published in production.

### CODE REVIEW

File: data-management-library/mysql/mysql-heatwave/4execute-sql-queries/execute-sql-queries.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This lab assumes you have:
 * You have enough privileges to use OCI
 * All previous labs successfully completed
 
-## **TASK 4**: Execute SQL Queries to See the difference
+## **Task 4**: Execute SQL Query to see the Performance Differences
```
Comment (anooshapilli) [inline data-management-library/mysql/mysql-heatwave/4execute-sql-queries/execute-sql-queries.md:21]: @jitender-git , Please rename Task 4 to Task 1

File: data-management-library/mysql/mysql-heatwave/3upload-data-into-heatwave/upload-data-into-heatwave.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ This lab assumes you have:
 * You have one Compute instance having <a href="https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html" target="\_blank">**MySQL Shell**</a> installed on it.
 * All previous labs successfully completed
 
-## **TASK 3**: Upload data into MySQL HeatWave Cluster Engine
+## **Task 3**: Upload data into MySQL HeatWave Cluster Engine
```
Comment (anooshapilli) [inline data-management-library/mysql/mysql-heatwave/3upload-data-into-heatwave/upload-data-into-heatwave.md:22]: @jitender-git , Please rename Task 3 to Task 1

File: data-management-library/mysql/mysql-heatwave/2enable-heatwave-cluster/enable-heatwave-cluster.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ This lab assumes you have:
 * You have enough privileges to use Oracle Cloud resources
 * All previous labs successfully completed
 
-## **TASK 2**: Enable HeatWave Cluster to MySQL Database System
+## **Task 2**: Enable HeatWave Cluster in MySQL Database System
```
Comment (anooshapilli) [inline data-management-library/mysql/mysql-heatwave/2enable-heatwave-cluster/enable-heatwave-cluster.md:21]: @jitender-git , Please rename Task 2 to Task 1

**Review by anooshapilli — COMMENTED** [review]
@jitender-git , Please remove the numbers before of the folder name. For example, Remove 1 before the folder 1create-mysql-database-system in the path - data-management-library/mysql/mysql-heatwave/1create-mysql-database-system/create-mysql-database-system.md

**Comment by jitender-git** [comment]
Hi @anooshapilli,

As suggested, I have modified the folder name (removed numbering) and changed Task. Please review and approve the changes.

Thanks,

**Comment by anooshapilli** [comment]
@jitender-git Please unbold the Tasks. i.e Task 1

**Comment by jitender-git** [comment]
Hi, done the changes. Please review and approve.

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**